packages/icons/src/utils/converter/generateComponent.ts

```diff
@@ -72,14 +72,20 @@ export const ${componentName} = ({
   semantic,
   inverted = false,
   svgProps,
+  "aria-label": ariaLabel,
+  "aria-labelledby": ariaLabelledBy,
+  "aria-describedby": ariaDescribedBy,
```

Comment:
We also need the `role` to be passed down to the SVG.

We should refactor this code out of `generateComponent`. Code in this template string get no proper validation/highlighting and will make the package size increase.

We could refactor it into an util, to split `others` props and `svgProps` - something like:

```tsx
const {svgProps, others} = getSvgProps(props);
```